appraisal.lic

```diff
@@ -54,18 +63,16 @@ class Appraisal
     DRSkill.getxp('Appraisal') < 30
   end
 
-  def assess_zills?
-    return false if DRSkill.getrank('Appraisal') >= 250
+  def assess_zills
+    return if DRSkill.getrank('Appraisal') >= 250
```

Comment:
Maybe it should echo to inform the user that they can remove this from their config, and/or need to find a new way to train Appraisal?